src/server/stream_family.cc

```diff
@@ -1005,6 +1007,16 @@ OpResult<ClaimInfo> OpClaim(const OpArgs& op_args, string_view key, const ClaimO
   ClaimInfo result;
   result.justid = (opts.flags & kClaimJustID);
 
+  // TODO: support replication via DF's propagation mechanism.
```

Comment:
@adiholden this is something I would like to consult you... in Redis' own implementation of xclaim (from function `xclaimCommand` of `t_stream.h`), it has an option of only propagate the last ID: by default it will call function `streamPropagateXCLAIM` which propagates the whole command. But if it detects that only last_id needs to be changed and other things such as the pending entry list does not need to be modified, then it willl set the `propagate_last_id` flag and do the following:

```
    if (propagate_last_id) {
        streamPropagateGroupID(c,c->argv[1],group,c->argv[2]);
        server.dirty++;
    }
    setDeferredArrayLen(c,arraylenptr,arraylen);
    preventCommandPropagation(c);
```

Is this kind of optimized behavior preserved in DF?

Reply:
So this is just an optimization? 
Lets talk tomorrow about replication and when we do change the original command in dragonfly to make replica consistent with master to make sure all stream commands work well

Reply:
> So this is just an optimization? Lets talk tomorrow about replication and when we do change the original command in dragonfly to make replica consistent with master to make sure all stream commands work well

wonderful.. exactly what i want to discuss tomorrow too!

Reply:
we had a discussion today, and we think it makes sense to not do the optimization Redis was doing regarding separately replicating last_id for now. The comments above in the code have thus been removed.